### PR TITLE
#178962516 estimate time of transaction

### PR DESCRIPTION
**Pull request by rango527**

### CODE REVIEW

File: packages/client/src/containers/landing-container.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import mixpanel from 'util/mixpanel';
 import ConnectWalletButton from 'components/connect-wallet-button';
 import { LiquidityContainer } from 'containers/liquidity-container';
 import { Box } from '@material-ui/core';
-import BananaHelp from 'components/banana-help/banana-help';
```
Comment (rootedbox) [inline packages/client/src/containers/landing-container.tsx:10]: please restore the banana-help

File: packages/client/src/containers/landing-container.tsx [hunk 2]
```diff
@@ -181,17 +206,14 @@ function LandingContainer({
                     {<ConnectWalletButton onClick={showWalletModal} />}
                 </div>
             </div>
-
```
Comment (rootedbox) [inline packages/client/src/containers/landing-container.tsx]: banana help is needed

**Review by rootedbox — APPROVED** [review]
looks good but can you restore banana help before merge. thanks 👍🏽

File: packages/client/src/containers/landing-container.tsx (hunk 1 above)
Comment (rango527) [reply packages/client/src/containers/landing-container.tsx:10]: Okay.

**Pull request merged**